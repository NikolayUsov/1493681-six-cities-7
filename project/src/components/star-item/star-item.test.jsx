import React from 'react';
import { render } from '@testing-library/react';
import StarItem from './star-item';

describe('Test star item component', () => {
  const fakeProps = {
    value: '2',
    title: 'badly',
    handleChangeRating: () =>{},
    currentValue: 3,
  };

  it('Should create snapshot correctly', () => {
    const { asFragment } = render(<StarItem {...fakeProps} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
