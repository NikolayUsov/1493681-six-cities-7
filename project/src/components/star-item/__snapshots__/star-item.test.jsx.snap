// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test star item component Should create snapshot correctly 1`] = `
<DocumentFragment>
  <input
    class="form__rating-input visually-hidden"
    data-testid="2-testid"
    id="2-stars"
    name="rating"
    type="radio"
    value="2"
  />
  <label
    class="reviews__rating-label form__rating-label"
    for="2-stars"
    title="badly"
  >
    <svg
      class="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlink:href="#icon-star"
      />
    </svg>
  </label>
</DocumentFragment>
`;
